system-addon/content-src/components/StartupOverlay/StartupOverlay.jsx

```diff
@@ -0,0 +1,86 @@
+import {FormattedMessage, injectIntl} from "react-intl";
+import {actionCreators as ac} from "common/Actions.jsm";
+import {connect} from "react-redux";
+import React from "react";
+
+export class _StartupOverlay extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.onInputChange = this.onInputChange.bind(this);
+    this.onSubmit = this.onSubmit.bind(this);
+    this.clickSkip = this.clickSkip.bind(this);
+    this.initScene = this.initScene.bind(this);
+
+    this.state = {emailInput: ""};
+  }
+
+  initScene() {
+    // Timeout to allow the scene to render once before attaching the attribute
+    // to trigger the animation.
+    setTimeout(() => {
+      this.setState({show: true});
+    }, 10);
+  }
+
+  removeOverlay() {
+    this.setState({signIn: true});
```

Comment:
Is it possible to re-use the `show` state here and just set it to false? Removing the overlay is basically just like not showing it anymore, so re-using the same `show` state seems to make sense

Reply:
I think so, will require some css changes to get the animations right